app/views/admin/chapters/index.html.haml

```diff
@@ -0,0 +1,45 @@
+%section#banner
+  .row
```

Comment:
a `.row` in foundation should contain column elements, so here there should be a `large-12.columns` to achieve correct alignment.